### PR TITLE
added get-created-posts from a logged in user

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

File: shared/components/myPage.js [hunk 1]
```diff
@@ -54,22 +54,15 @@ class MyPage extends Component {
     this.renderTabViewContents = this.renderTabViewContents.bind(this);
     this.renderTabView = this.renderTabView.bind(this);
     getAccessToken().then((accessToken) => {
-      console.log(accessToken);
```
Comment (yuda1124) [inline shared/components/myPage.js]: good!!!

File: shared/components/myPage.js [hunk 2]
```diff
@@ -90,6 +83,30 @@ class MyPage extends Component {
       }
     });
     this.props.getSavedPosts();
+    // const address = 'http://goober.herokuapp.com/api/users/createdposts';
+    const address = `${HTTPS}${SERVER_ADDR}${ENDPOINT_CREATEDPOST}`;
+    getAccessToken().then((accessToken) => {
+      console.log(address);
+      console.log(accessToken);
```
Comment (yuda1124) [inline shared/components/myPage.js]: console

File: shared/reducers/index.js [hunk 3]
```diff
@@ -5,12 +5,14 @@ import fluxReducer from './flux';
...
 
 export default combineReducers({
   map: mapReducer,
   list: listReducer,
   flux: fluxReducer,
   form: formReducer,
   myPage: myPageReducer,
-  auth: authReducer
+  auth: authReducer,
+  user: userReducer
```
Comment (yuda1124) [inline shared/reducers/index.js]: why don't you use myPageReducer?

File: shared/components/Card.js [hunk 4]
```diff
@@ -23,7 +23,11 @@ const styles = {
     flexDirection: 'column',
     borderBottomWidth: 1,
     borderBottomColor: '#e7e7e7',
-    elevation: 5
+    ...Platform.select({
```
Comment (ochanje210) [inline shared/components/Card.js]: yes it is

**Comment by yuda1124** [comment]
LGTM

**Pull request merged**